README.md

```diff
@@ -1,80 +1,125 @@
-# Getting Started with Create React App
+# react_group_project
```

Comment:
- [ ] Please remove the name of the project from here, because you already have added the project name just after `table of contents`. Kindly avoid redundancy in your documentation because it makes your `README` unprofessional.